### PR TITLE
arm32 and protocol v1 support, jsclient transmit method added. 

### PR DESCRIPTION
**Pull request by ace-dog**
fix[arm32] change position of counter in Client so run on arm32
feat[Protocol v1] add response to protocol v1
feat[jsclient] add Transmit method from jsclient

### CODE REVIEW

File: scclient/jsclient.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package scclient
+
+import (
+	"github.com/sacOO7/socketcluster-client-go/scclient/models"
+	"github.com/sacOO7/socketcluster-client-go/scclient/utils"
+)
+
+func (client *Client) Transmit(eventName string, data interface{}) {
```
Comment (sacOO7) [inline scclient/jsclient.go:8]: We already have a `emit` method. It has same impl as transmit

Comment (ace-dog) [reply scclient/jsclient.go:8]: yes, and the main change is no cid.

Comment (sacOO7) [reply scclient/jsclient.go:8]: OKAY

**Comment by sacOO7** [comment]
bTW @ace-dog feel free to open more PR's if you feel there is chance of improvement 👍

**Pull request merged**